docs/OTP2-MigrationGuide.md

```diff
@@ -25,22 +25,50 @@ you want into a single instance of OTP2.
 
 ## Build config
 
+New parameters:
+
+ - `transitServiceStart` Limit the import of transit services to the given *start* date. Default: `-P1Y`
+ - `transitServiceEnd` Limit the import of transit services to the given *end* date. *Inclusive*. Default: `P3Y`
+
+
 These properties changed names from:
 
  - `htmlAnnotations` to `dataImportReport`
  - `maxHtmlAnnotationsPerFile` to `maxDataImportIssuesPerFile`
  - `boardTimes` to `routingDefaults.boardSlackByMode`
  - `alightTimes` to `routingDefaults.alightSlackByMode`
  
-These parameters is no longer supported:
-
- - `stopClusterMode` - TODO OTP2 Why? Old options: `proximity`, `parentStation`
+These parameters are no longer supported:
+
+ - `stopClusterMode` OTP2 link stops and stations based on the GTFS/NeTEx input. There is no
+         automatic linking any more. If this is needed, feel free to port the OTP1 logic into 
+         a Sandbox feature. The OTP1 options are: `proximity`, `parentStation`.
+ - `parentStopLinking` OTP2 link stops and stations based on the GTFS/NeTEx input, but there is no 
```

Comment:
I think concepts like "there is no such thing in the street graph" will not be easily understood by the average person configuring OTP2. Again here I think we should say what exactly it means that they are "linked" based on the input. Later in the same paragraph transfers are mentioned; could we replace "links" with "creates transfers between"?